packages/xchain-thorchain/src/client.ts

```diff
@@ -782,6 +782,7 @@ class Client extends BaseXChainClient implements ThorchainClient, XChainClient {
       chainId: this.getChainId(),
       nodeUrl: this.getClientUrl().node,
     })
```

Comment:
needs change log filled in 

revert current package back to version": "0.28.8",

`yarn updatePackages patch Update "Added prepareTx() to client" thorchain`

what this will do is update the version and fill the change log in 

then run `yarn updateDeps 0.28.9 thorchain` this will bump the version on any other package in the library that depends on it.